mdapi/services/appviews.py

```diff
@@ -25,20 +25,20 @@
 
 from aiohttp.web import FileResponse, HTTPBadRequest, json_response
 
-from mdapi.confdata.servlogr import logrobjc
-from mdapi.confdata.standard import DB_FOLDER
+from mdapi.confdata import servlogr, standard
 from mdapi.services import _expand_package_info, _get_changelog, _get_files, _get_package
 
 homepage = os.path.join(os.path.dirname(os.path.abspath(__file__)), "homepage.html")
 
 
 async def index(rqst):
-    logrobjc.info("index %s" % rqst)
+    servlogr.logrobjc.info("index %s" % rqst)
     return FileResponse(homepage)
 
 
 async def get_pkg(rqst):
-    logrobjc.info("get_pkg %s" % rqst)
+    servlogr.logrobjc.info("get_pkg %s" % rqst)
+    print(standard.DB_FOLDER, "BEQUD")
```

Comment:
This looks like a leftover from debug, no?

Reply:
Fixed with https://github.com/t0xic0der/mdapi/pull/36/commits/51d007e64185a5bf366e77b6f35569e003c37a3f